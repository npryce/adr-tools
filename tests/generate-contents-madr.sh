adr init docs madr
adr new First Decision
adr new Second Decision
adr generate toc
