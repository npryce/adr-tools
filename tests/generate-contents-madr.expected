adr init docs madr
docs/0000-record-architecture-decisions.md
adr new First Decision
docs/0001-first-decision.md
adr new Second Decision
docs/0002-second-decision.md
adr generate toc
# Architecture Decision Records

* [Use Markdown Architectural Decision Records](0000-record-architecture-decisions.md)
* [First Decision](0001-first-decision.md)
* [Second Decision](0002-second-decision.md)
